resources/definitions/os_detection/endace.yaml

```diff
@@ -0,0 +1,15 @@
+os: endace
```

Comment:
Looks like this OS should be called `osm`. These detections are for the OS running rather than the company / vendor name.

Reply:
@peter-endace Tagging in case you missed the above